src/NuGet.Services.AzureSearch/AzureSearchTelemetryService.cs

```diff
@@ -346,14 +346,14 @@ public void TrackDownloadSetComparison(int oldCount, int newCount, int changeCou
                 });
         }
 
-        public void TrackAuxiliary2AzureSearchCompleted(bool success, TimeSpan elapsed)
+        public void TrackAuxiliary2AzureSearchCompleted(JobOutcome outcome, TimeSpan elapsed)
         {
             _telemetryClient.TrackMetric(
                 Prefix + "Auxiliary2AzureSearchCompletedSeconds",
                 elapsed.TotalSeconds,
                 new Dictionary<string, string>
                 {
-                    { "Success", success.ToString() },
+                    { "Outcome", outcome.ToString() },
```

Comment:
why not add the string value instead of the enum in the metrics? Its more readable, otherwise one has to know the translation of enums to understand if the outcome succeeded or failed.

Reply:
This will result in a readable string. Example: https://dotnetfiddle.net/aynBJL
```
Outcome: NoOp
```

Reply:
ah, my bad.. default prints string. `ToString("d")` is not the default.

https://docs.microsoft.com/en-us/dotnet/api/system.enum.tostring?view=netframework-4.8